### DISCUSSION TITLE
Raspberry Pi 3B instead of B+

### DISCUSSION THREAD

**Discussion by germanviscuso** [frntc/Sidekick64#22] [Q&A]
Does the Sidekick64 work with a plain Raspberry Pi 3B rather than the B+ ? That's the one I have at home and the 3B+ is difficult to source right now.

**Comment by frntc** [comment]
The 3B (without plus) behaves a bit differently than the B+ in terms of timings/caching (as far as I can observe). Some 3B, however, use the same/similar SoC as the 3B+, and might work after manually adjusting the timings. But this is a bit awkward and thus it's currently not officially supported. As a bright spot: there will be an updated version of the software which also runs on a RPi Zero 2, I might test that one on a 3B as well...

**Comment by pigravity** [comment]
Wow, found this project few days ago, pure awesomness :D Completed most of chips, smds... Ofc... there is NO Pi available around :D That bothers me most. Would be good if it could be figured out to work with diff boards aswel.

**Comment by MartinGoodwell** [comment]
For me it works with a 3B. Flawlessly

**Reply by pigravity** [reply]
yea gotta get 2nd hand one and we will see :D cheers

**Comment by slesinger** [comment]
I am using RPI 3B because I cannot find 3B+ nor Zero2w for less than a ridiculous price. Almost everything works but not 100% (cartridge fastloaders are a problem). Can you please point me into how to adjust the timings that you reference in a post above? Do I need to compile the code? Do I need to measure timing on some pins using oscilloscope? Thank you

**Comment by frntc** [comment]
There were people using a 3B on forum64.de: https://www.forum64.de/index.php?thread/101221-sidekick64-aufbau-probleme-l%C3%B6sungen/&postID=1630331#post1630331
... but this post is really old.

It's not so trivial to get the timings working (and I don't have a 3B to teset), and the following is just a bunch of observatiions:
- the main clock is not the most important (if possible choose one of the profiles with 1.4GHz or so -- should work on the 3B, in particular when it's a new stepping
- try experimenting with over voltage. This setting has large influence on the GPIO speed/bus timings
- SDRAM and GPU clock only influence the SK64 performance indirectly as the affect cache prefetching etc
